lib/model/CMetricModel.cc

```diff
@@ -536,6 +536,10 @@ void CMetricModel::fill(model_t::EFeature feature,
 
     std::size_t dimension{model_t::dimension(feature)};
     const TFeatureData* data{this->featureData(feature, pid, bucketTime)};
+    if (data == nullptr) {
+        LOG_TRACE(<< "data unexpectedly null");
+        return;
+    }
```

Comment:
Model being null is also unchecked below.

Reply:
Also, as with other cases I think if this fails we need to short circuit in calling code.